### DISCUSSION TITLE
4-body interactions between 88Sr

### DISCUSSION THREAD

**Discussion by PedroIldefonso** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#151] [General]
I have recently started using Arc for a project that involves the study of interactions between Rydberg stated of 88Sr. I have managed to simulate the interaction between a pair of these atoms, however as far as I am aware there is no way of simulating the interaction between four of these at the same time.

I am very new to this field of research and most of my colleagues do not have much insight on this field of atomic physics. So, what I wanted to ask was if you have any insight on how this might be done and if your team is actually working on something like this or not.

**Comment by nikolasibalic** [comment]
ARC does not diagonalize interactions for more than 2 atoms in current version (any updates/PR regarding this are welcome!).

But you can use ARC to construct such calculation. You need to select N-atom state basis (instead of just pair-state basis), cut it in the space such that you consider only certain energy range of N-body state defects (similarly as we do for pair-state basis calculations), fill the interaction matrix with dipolar coupling terms and corresponding energy defects (from methods in ARC), and then diagonalize it. You can create this based on pair-state interactions.

I would assume that in most cases two-body terms would dominate, and then sum of such terms obtained from pair-state calculations should be sufficiently good approximation.

However, there can be situations where the first dominant term does not include exchange between pairs of atoms (like van der Waals interaction), but between three or more atoms (see [three](https://doi.org/10.1038/ncomms9173) and [four](https://doi.org/10.1103/PhysRevLett.108.023005) body Foster resonances). Then diagonalizing N-body interaction matrix is must. An example where 3-body interaction term is dominant is given at [T. Pohl and P. R. Berman, Phys. Rev. Lett. 102, 013004](https://doi.org/10.1103/PhysRevLett.102.013004). 

Of course parameter space describing positions of N-atoms is larger than that of descibing position of 2 atoms (where distance and angle to z axis suffice). That's one of the reasons why exploring such interactions in detail can be complicated.

I hope this helps!
